node/impl/client/client.go

```diff
@@ -35,8 +34,11 @@ import (
 	"github.com/filecoin-project/lotus/node/impl/full"
 	"github.com/filecoin-project/lotus/node/impl/paych"
 	"github.com/filecoin-project/lotus/node/modules/dtypes"
+	"github.com/filecoin-project/specs-actors/actors/abi"
 )
 
+const dealStartBuffer abi.ChainEpoch 100
```

Comment:
This is quite low, bit over 1h. It's not really possible to seal a sector in that time. Will bump in specs-actors branch.